### DISCUSSION TITLE
LDAP or LDAPS Authentification

### DISCUSSION THREAD

**Discussion by DarthDensus** [NginxProxyManager/nginx-proxy-manager#2879] [Ideas]
Is there any feature plans to implement usermanagment over LDAP. I know the primary propose was to develop an revproxy for personal  or private networks. We use it in our company and we are very exited about the simply configuration of proxy hosts. We like the idea of reducing administartively work with simple and smart applications.

**Comment by Wadera** [comment]
https://github.com/NginxProxyManager/nginx-proxy-manager/pull/5345 :)

**Comment by Stephan-P** [comment]
Very interesting as I am currently struggling with making NPM, LLDAP and TinyAuth work together.

- Would this be coming to a next release?
- Is (preliminary) documentation available, including examples of co-op with other containers and forwarding of credentials?

(Excuse me if I'm using wrong terminology here. I appreciate the concept of centralised management of users, credentials and groups, but have a hard time understanding the inner workings of it and what's going on between docket containers)
